docs/reference/api/connectivity/nfc/NDEF.md

```diff
@@ -0,0 +1,106 @@
+### NDEF API
+
+Mbed OS provide this API to construct and parse NDEF messages, which is the common data format exchange for NFC messages.
+
+An NDEF message is a collection of separate NDEF records. Each of these records is defined by a type, such as URI, Mime and Text, that identifies a payload and what the record contains.
+
+#### Parsing
+
+The class `mbed::nfc::ndef::MessageParser` parses a buffer of data in input and produces parsing events that are forwarded to its delegate. To help you, we offer a more integrated parser (`mbed::nfc::ndef::common::SimpleMessageParser`) that parses well known NFC types records, such as Text, URI or Mime records, and produces usable objects out of the box.
+
+```
+#include "nfc/ndef/common/SimpleMessageParser.h"
+
+using mbed::nfc::ndef::Record;
+using mbed::nfc::ndef::RecordType;
+using mbed::nfc::ndef::RecordID;
+using mbed::nfc::ndef::MessageParser;
+using mbed::nfc::ndef::common::Text;
+using mbed::nfc::ndef::common::URI;
+using mbed::nfc::ndef::common::Mime;
+
+using mbed::nfc::ndef::common::SimpleMessageParser;
+
+void parse_ndef_message(const Span<const uint8_t> &buffer) {
+    SimpleMessageParser parser;
```

Comment:
What's the purpose of these functions?
Are they included in Mbed OS, or something that a developer will need to copy & paste in their application?

Reply:
The line above the code it is indicated that `mbed::nfc::ndef::common::SimpleMessageParser` is part of the API provided. 

The code just demonstrate how to use this class to parse an NDEF message.